lms/djangoapps/discussion/toggles.py

```diff
@@ -13,3 +13,12 @@
 # .. toggle_creation_date: 2021-11-05
 # .. toggle_target_removal_date: 2022-03-05
 ENABLE_DISCUSSIONS_MFE = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'enable_discussions_mfe', __name__)
+
+# .. toggle_name: discussions.enable_learners_tab_in_discussions_mfe
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to enable learners tab in the new MFE experience for discussions
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2022-02-21
+# .. toggle_target_removal_date: ?
```

Comment:
@xitij2000 I assume that this is going to be a temporary flag. What should be the `togle_target_removal_date`?

Reply:
@Cup0fCoffee we can keep it for 3 months and then revisit if needed.